plugins/core/core/src/software/aws/toolkits/core/utils/RemoteResourceResolver.kt

```diff
@@ -32,9 +32,11 @@ class DefaultRemoteResourceResolver(
     private fun internalResolve(resource: RemoteResource): Path {
         val expectedLocation = cacheBasePath.resolve(resource.name)
         val current = expectedLocation.existsOrNull()
-        if (current != null && !isExpired(current, resource)) {
-            LOG.debug { "Existing file ($current) for ${resource.name} is present and not expired - using it." }
-            return current
+        if (resource.name != "notifications.json") {
```

Comment:
@rli This feels hacky, but its one way to get around this issue, the other way I think we could maybe do this is to just set the expiry time to a very short window so that the file is always seen as expired by the time polling makes this call to the endpoint